server/src/controllers/employer/getAllEmployers.js

```diff
@@ -16,12 +17,31 @@ const getAllEmployersRequestHandler = async (req, res) => {
       endDateAdded,
       ownerId,
       postalCode,
+      contactName,
     } = req.query;
 
     const query = {};
     if (employerName) {
       query.name = { [Op.like]: `%${employerName}%` };
     }
+
+    let employerContactIds = [];
+    if (contactName) {
+      const employerContacts = await EmployerContact.findAll({
+        where: {
+          name: { [Op.like]: `%${contactName}%` },
+        },
+      });
+      employerContactIds = employerContacts.map((contact) => contact.id);
```

Comment:
Instead of extracting the id (which returns the employer contact id), we should be extracting the employer attribute here (the id of the associated employer).